src/apps/inventory/migrations/0002_product_seller.py

```diff
@@ -1,8 +1,8 @@
 # Generated by Django 4.2.5 on 2023-09-29 13:00
 
+import django.db.models.deletion
```

Comment:
don't change migrations pls

Reply:
i think imports in migrations should be sorted as well